### ISSUE TITLE
uid must be an unsigned int on npm install typings --global 

### ISSUE THREAD

**Issue by tggsilva** [npm/npm#14077]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [X] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [ ] Other (_see below for feature requests_):

#### What's going wrong?
I'm getting * uid must be an unsigned int * when trying to install the npm install typings --global for typescript

#### How can the CLI team reproduce the problem?
Well I just tried to install the typings

### supporting information:

 - `npm -v` prints: v3.10.8
 - `node -v` prints: v6.6.0
 - `npm config get registry` prints: https://registry.npmjs.org/
 - Windows, OS X, or Linux?: OS X [Sierra: 10.12 (16A323)]
 - Issue Details
````
npm ERR! uid must be an unsigned int
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
npm ERR! Darwin 16.0.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "typings" "--global"
npm ERR! node v6.6.0
npm ERR! npm  v3.10.8
````
- npm-debug.log: 
[npm-debug.txt](https://github.com/npm/npm/files/492009/npm-debug.txt)

#### What's the feature?
Trying to use typescript

#### What problem is the feature intended to solve?
typings

#### Is the absence of this feature blocking you or your team? If so, how?
I can't use typescript because it does not recognize the modules

#### Is this feature similar to an existing feature in another tool?
N/A

#### Is this a feature you're prepared to implement, with support from the npm CLI team?
I didn't understand the question

**Comment by mlkb** [comment]
Having the same problem. 

npm ERR! Darwin 16.0.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "-g" "n"
npm ERR! node v0.10.34
npm ERR! npm  v3.10.8

npm ERR! uid must be an unsigned int
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/name/Fold/npm-debug.log

**Comment by KenanY** [comment]
Looks like a duplicate of https://github.com/npm/npm/issues/13918?

**Comment by hakimCZE** [comment]
I am having the problem when tryin to install `babel-cli` globaly. 
Running:
`Mac OS El Captain`
`npm -v 3.10.8`
`node -v v6.7.0`

**Comment by ciwyaitd** [comment]
I have the same problem.
Mac OS El Captain
npm -v 3.10.8
node -v v6.2.0

**Comment by hakimCZE** [comment]
So, the solution for me, was to uninstall node.js via homebrew, install it again (version 6.9.1). After that, global installing works again. Npm version is still 3.10.8

Disclaimer, when I tried to reinstall only npm, it stoped workin completely. Also, this happend when I did not removed all node cells. 

To install specific version from tap, I followd instructions in this post, hope it will help.
http://effectif.com/mac-os-x/installing-specific-version-of-homebrew-formula
